src/main/resources/script_templates/Examples/Blur_Image.scala

```diff
@@ -0,0 +1,26 @@
+// @Dataset inputImg
+// @OpService ops
+// @OUTPUT Dataset(label="Blurred") blurredImg
+
+// A simple Scala script that blurs an input image with
+// a Gaussian filter. Note that you must have an image
+// open to run this script.
+// It is the duty of the scripting framework to bind
+// the input image and then display the blurred output
+// image. It also binds the ImageJ OpService for filtering.
+
+import net.imagej.ops.OpService
+import net.imglib2.img.Img
+import net.imglib2.`type`.numeric.RealType
+
+// Scala is a compiled language with a very strict/powerful
+// type system. We need to cast any parameters that are bound
```

Comment:
Too bad... it would be nice if the script engine could tell Scala the intended types somehow, since we do know them at runtime. But awesome that you got this working.